deploy/conf.sh

```diff
@@ -0,0 +1,5 @@
+export PGHOST=rankdb.coomuod9pyh1.us-west-2.rds.amazonaws.com
+export PGPORT=5432
+export PGDATABASE=rankdb
+export PGUSER=rankuser
+export PGPASSWORD=rankpassword
```

Comment:
it's in db/terraform.tfstate anyways, made a task to hide this later.

 world, if you're reading this, please fuck our database up. I don't care.